cluster/config-defaults.yaml

```diff
@@ -96,6 +91,17 @@ horizontal_pod_autoscaler_sync_period: "30s"
 horizontal_pod_autoscaler_tolerance: "0.1"
 horizontal_pod_autoscaler_upscale_delay: "3m0s"
 
+# skipper ingress settings
+skipper_ingress_target_average_utilization_cpu: "100"
+skipper_ingress_target_average_utilization_memory: "80"
+skipper_ingress_max_replicas: "10"
```

Comment:
This would be a max of `30`, maybe we want to use `5` as the max to limit it to `15` as we talked about, to prevent scaling up too much?

/cc @aermakov-zalando since you made a point about this in our meeting :)

Reply:
Our internal ticket, which we agreed on in the meeting with everyone interested was 10 per AZ.